### PR TITLE
Add SonarCloud scan to CI workflow

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=City-of-Helsinki_linkedevents
+sonar.organization=city-of-helsinki
```
Comment (charn) [inline sonar-project.properties:2]: We could/should add `sonar.python.version=3.9` here?

Comment (tuomas777) [reply sonar-project.properties:2]: Yes 👍

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Review by voneiden — APPROVED** [review]
Neat

**Pull request merged**